test/MediatR.Tests/PipelineTests.cs

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
I kinda feel that this whole file should be reverted, bar the removing of the namespace - just inconsistent formatting

Reply:
I rolled it back. On save, my IDE (Rider) formats all open files to my coding standards.

Reply:
So basically I need an `.editorconfig`